carbon-design-system/ibm-security
# Data decorator panel root

## Proposed changes

- Adding DataDecorator `rootNode` prop proxy to PanelV2 component
- Adding DataDecorator `focusTrap` prop proxy to PanelV2 component
- Adding PanelV2 `rootNode` prop proxy to the Portal component

## Testing instructions

- Pass a rootNode (HTMLElement) to DataDecorator and ensure the panel is rendered within that node
- Disable the focus trap for the DataDecorator and verify it is no longer active
